docs/SMG/08-deployment.md

```diff
@@ -9,7 +24,31 @@ ssh -i {certificate pem file here} vectoratlasadmin@20.87.47.170
 
 The certificate and other credentials are stored in the Hybrid Intelligence Notes Project Database under technical notes for the project. ICIPE ICT also maintains access to the system.
 
-## Configuring the base machine
+## Manually updating the system
+If a deployment is more complex than is possible with the automated deployment then log into the virtual machine and run:
+```
+cd ~/vector-atlas/vectoratlas-software-code/src/Docker
+docker-compose down
+git fetch
+git checkout --force [sha of the desired commit]
+
+cd ..
+chmod +x buildVersionFiles.sh
+./buildVersionFiles.sh
+
+cd Docker
+docker-compose build
+docker-compose up --detach
+```
+
+### Running database migrations
```

Comment:
Do we still need this now it's part of the stack?

Reply:
I've put in some extra text to make it clearer that this normally happens as part of the docker compose stack but said that if we need to run it manually then this is the command.